time-elements.js

```diff
@@ -587,6 +589,33 @@
     }
   }
 
+  // Private: Format a time according to the `timezone` attribute values.
+  //
+  // el - The local-time element to format.
+  //
+  // Returns a timezone String or null if no timezone formats are provided.
+  function formatTimezone(el) {
+    // retrieve format settings from attributes
+    var options = {
+      timezone: el.getAttribute('timezone')
+    };
+
+    // remove unset format attributes
+    for (var opt in options) {
+      if (!options[opt]) {
+        delete options[opt];
+      }
+    }
+
+    // no timezone format attributes provided
+    if (Object.keys(options).length === 0) {
+      return;
+    }
+
+    var match = el._date.toString().match(/\(([^)]+)\)/);
+    return match ? match[1] : '';
+  }
+
```

Comment:
The [Intl.DateTimeFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DateTimeFormat) supports a `timeZoneName` option, so we would need to expose that with a `timeZoneName` attribute on the custom element and [pass it into the formatter](https://github.com/github/time-elements/blob/da6eb9f6cfbdb43bb45ef8ff23e2515667ae1214/time-elements.js#L564). That will avoid regular expression parsing of unknown `toString()` text.